src/main/kotlin/com/example/bankAccount/domain/Account.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.bankAccount.domain
+
+import org.iban4j.Iban
+import java.math.BigDecimal
+
+data class Account(
+    val iban: String = Iban.random().toString(),
+    val firstName: String,
+    val lastName: String,
+    val balance: BigDecimal = BigDecimal.ZERO,
+    val transactions: MutableList<Transaction> = ArrayList()
```

Comment:
```suggestion
    val transactions: List<Transaction> = emptyList()
```

En Kotlin, on préfèrera toujours les collections immutables aux collections mutables.